python/delta-kernel-rust-sharing-wrapper/Cargo.toml

```
@@ -11,7 +11,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 arrow = { version = "53.3.0", features = ["pyarrow"] }
-delta_kernel = {version = "0.5", features = ["cloud", "default", "default-engine"]}
+delta_kernel = { git = "https://github.com/delta-io/delta-kernel-rs.git", rev = "be1453f", features = ["cloud", "default", "default-engine"]}
```

default features are empty 
```suggestion
delta_kernel = { git = "https://github.com/delta-io/delta-kernel-rs.git", rev = "be1453f", features = ["cloud", "default-engine"]}
```